app/subscriber/src/features/content/view-content/ViewContent.tsx

```diff
@@ -153,7 +188,33 @@ export const ViewContent: React.FC = () => {
         >
           <p>{parse(content?.summary ?? '')}</p>
         </Show>
+        <Show visible={content?.contentType === ContentTypeName.Snippet}>
+          <Button
+            onClick={() => handleTranscribe()}
+            variant={ButtonVariant.action}
+            className="transcribe-button"
+            disabled={
+              (!!content?.fileReferences && !content?.fileReferences.length) ||
```

Comment:
Button disabled when there is no video attached to a snippet, or if the transcription has already been completed,